### DISCUSSION TITLE
小玉換臉事件的兇器—deepfake

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#977] [閒聊八卦]
前陣子鬧得沸沸揚揚的的小玉事件，讓大家正視換臉技術和隱私權相關的問題。
Deepfake是個雙面刃，由於使用者動機、目的的不同，不能斷定deepfake就是有害隱私權的工具。
所謂知己知彼，百戰百勝，若想好好防禦deepfake帶來的影響，我們勢必要更了解deepfake的運作與功能。

![921B7192-09AA-4F15-A80E-26B1669168B1](https://user-images.githubusercontent.com/113414274/210716390-36f54e89-0b2d-4050-89f0-882523efc402.jpeg)
https://pansci.asia/archives/342432

**Comment by harrylee1971** [comment]
補充一下，deepfake的背後原理是線性代數的轉換函式

**Comment by ghost** [comment]
網址好像有問題喔~
換臉技術的發展提供一種改變面容的預覽方法，對於整容行業很有用處，但以這種方法偽造成他人行騙，甚至是以此盈利，不僅是不道德的行為，還有可能對面容的原主人造成傷害

**Comment by a45320614** [comment]
面貌是人類能夠相互辨識的其中一個主要特徵，如果能夠隨意替換自己的面貌，將會出現許多詐騙、妨害名譽等等的犯罪事件，因此我個人認為政府應該要制定相關的法規來避免此類事件的發生，不知不覺中被一個陌生人使用自己的臉合成出一些圖片或影片真的是蠻噁心的。

**Comment by HsuChen88** [comment]
換臉的議題也牽涉到數位安全，因為現在很多認證是使用臉部辨識，甚至Deepfake能做出其他部位的話，其他的認證方式也會遭到假冒。

**Comment by pierre0210** [comment]
deepfake其實不只有這些負面功能，在一些正經的(?藝術領域也有其貢獻，像在Kendrick Lamar去年發布the heart part 5的MV中就有用deepfake技術致敬一些有名的歌手或演員，還是端看技術的使用者如何抉擇。
https://www.youtube.com/watch?v=uAPUkgeiFVY
